assets/locales/fr.po

```diff
@@ -1265,3 +1265,21 @@ msgstr ""
 
 msgid "File copy Suffix"
 msgstr "copie"
+
+msgid "Install New App Title"
+msgstr "Installez la nouvelle app Cozy !"
+
+msgid "Install App Unavailable"
+msgstr "Cette version de %s n'est désormais plus prise en charge ."
```

Comment:
No space before the dot

Reply:
For information this space came from a recommendation from @clochix but I will change.